docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -304,3 +304,23 @@ Time until expiration for tokens which are renewed. Should have a value higher t
 ### lease_renewal_increment
 
 New duration for renewed tokens. Vault may be configured to ignore this value and impose a stricter limit.
+
+## [security.egress]
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+### host_deny_list
+
+A list of hostnames or IP addresses separated by spaces for which requests will be blocked.
```

Comment:
```suggestion
A list of hostnames or IP addresses separated by spaces for which requests are blocked.
```

Reply:
The default in tech writing is to write in the present tense for general things that now and in general. Unless there is a specific event that must happen in the future, use present tense.

Reply:
Right, thanks.